test/functional-portable/dynamicrp/noncloud/resources/testdata/usertypealpha.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test.Resources
+types:
+  userTypeAlpha:
+    capabilities: ["SupportsRecipes"]
+    apiVersions:
+      "2023-10-01-preview":
+        schema:
+          type: "object"
+          properties:
+            application:
+              type: string
+              description: The resource ID of the application.
+            environment:
+              type: string
+              description: The resource ID of the environment.
+            capacity:
+              type: string
+              description: "The capacity of the API. Valid values: 'S', 'M', 'L'."
+            provisioningState:
```

Comment:
What will happen if a user doesn't specify `provisioningState` in the resource schema?